### ISSUE TITLE
place all [non-error] console.log()'s behind a debug option?

### ISSUE THREAD

**Issue by ffd8** [hydra-synth/hydra-synth#160]
hydra-synth [console.log's the canvas width](https://github.com/hydra-synth/hydra-synth/blob/a51741cf79a11e8fbf38abea457b561ea803f844/src/hydra-synth.js#L175) every time one adjusts `setResolution()` among a few other other spots that are not errors. On purpose or should have been commented out? While console.log() for any errors is of course necessary (riskyBiz mode could also supress them?) – what if [all others](https://github.com/search?q=repo%3Ahydra-synth%2Fhydra-synth%20console.log\(&type=code) sat behind the [API opts](https://github.com/hydra-synth/hydra-synth?tab=readme-ov-file#api) when initializing hydra with something like:

```js
{
  // ...
  debug: false, // console.log details from hydra as it takes place
}
```
*not sure if it would be best true or false by default? more beginner friendly info or confusing?*

**Comment by geikha** [comment]
I like having this outputted. Maybe this `debug` toggle could be globally accesible as part of the API and more information can be added.

**Comment by ojack** [comment]
I agree these should be suppressed by default with a debug toggle of some sort.

**Comment by bodobraegger** [comment]
working on this!

**Comment by ojack** [comment]
@bodobraegger that's great to hear, thank you!
